src/main/java/seedu/address/model/client/policy/CustomDate.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.model.client.policy;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * A class that represents a date in the format of dd.mm.yyyy.
+ */
+public class CustomDate {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "date should only contain numbers, in the format of dd.mm.yyyy";
+
+    public static final String VALIDATION_REGEX = "^\\d{2}\\.\\d{2}\\.\\d{4}$";
```

Comment:
If we were to follow this particular REGEX, would we be able to catch exceptions if the user passes in an invalid date?

Reply:
I believe so. The `CustomDate` class will handle correct format, but wrong date errors.